Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.7.0'
-
+gem 'rake'
```

Comment:
@h-m-m , i think this is a reasonable fix for [this CircleCI failure](https://app.circleci.com/pipelines/github/maebeale/mutual-aid/189/workflows/9b061ef6-7065-46c5-bbc6-100ac979764c/jobs/719), but let me know if you have any concerns about it.

Reply:
The problem was that the checked-in Gemfile.lock didn't match the Gemfile.lock after doing `bundle install`. I think I can adjust the CircleCI config that makes troubleshooting this easier. Can you push up one of the commits that failed to its own branch so I can test the improved config works?

Reply:
This change should not be needed. If you revert it and then cherry-pick https://github.com/maebeale/mutual-aid/commit/1188a0e177347caf077859344e38bbd17febe3eb you should get a much more useful error message if the bundling goes wrong

Reply:
Thanks for looking into this @h-m-m !
As you predicted, once i corrected the lock file, it works [fine](https://app.circleci.com/pipelines/github/maebeale/mutual-aid/223/workflows/357e2fb2-b5b0-4a1b-a1b9-d1032c60c9f4) w/o needing `rake` to be explicitly defined.
I've dropped that commit.